### ISSUE TITLE
charites doesn't respond to favicon.ico requests

### ISSUE THREAD

**Issue by pnorman** [unvt/charites#116]
**Describe the bug**

Browsers will automatically request `/favicon.ico` from servers sometimes. When this happens, charites doesn't respond, leaving the browser download as pending, causing blocking of other downloads, resulting in the entire page not loading.

```
$ curl -v http://127.0.0.1:8080/favicon.ico
*   Trying 127.0.0.1:8080...
* Connected to 127.0.0.1 (127.0.0.1) port 8080 (#0)
> GET /favicon.ico HTTP/1.1
> Host: 127.0.0.1:8080
> User-Agent: curl/7.74.0
> Accept: */*
>
```
This never completes.

**Expected behavior**
favicon.ico returns an empty response, an error, or anything.

**Charites Version:**

- Charites version: 0.1.4

**Desktop (please complete the following information):**

- OS: Debian

**Comment by pnorman** [comment]
Looks like f56753ab29dd76d7296892c3c378a6cd168e92f3 fixes this.

Can we have a new release to get the fix out?

**Comment by keichan34** [comment]
@pnorman Sorry for the late response -- 0.2.0 was released with f56753ab29dd76d7296892c3c378a6cd168e92f3 a few days ago.
